lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
1 won't work since symbols don't transfer well over vm proxy right now, which would make it impossible to do in window.

I guess 2 is true, but since the impl classes are the thing you actually maintain, I think it should kind of resemble what it's used for instead of looking like random util classes.